json/embeddings/embeddings-docs.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema#",
+  "$id": "https://impresso.github.io/impresso-schemas/json/embeddings/embeddings-docs.schema.json",
+  "title": "Document Embeddings JSON Schema",
+  "description": "A representation for the vector embedding representation of content items.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Unique identifier for the content item.",
+      "examples": ["actionfem-1940-01-08-a-i0001"],
+      "pattern": "^(.*)$"
+    },
+    "embedding": {
+      "oneOf": [
```

Comment:
I think there should be a flag somewhere to indicate whether the `embedding` is a list or a list of list, to avoid having to parse it to know it - e.g. :

```
  "multiple_embeddings | chuncked embeddings": {
      "type": "boolean",
      "description": "Indicates whether a content item has multiple or a single embedding."
    },
```